Common/ArmEmitter.cpp

```diff
@@ -268,7 +268,7 @@ bool ARMXEmitter::TryANDI2R(ARMReg rd, ARMReg rs, u32 val)
 		}
 		return true;
 	} else {
-#ifdef HAVE_ARMV7
+#if PPSSPP_ARCH(ARMV7)
```

Comment:
Yeah, this can be a runtime check. Come to think of it, emitters are kind of a special case that while they are arch specific, it makes sense to run them on other archs for cross compile or testing reasons. Therefore, we're probably best off building all the emitters on all the platforms and letting the linker sort it out. The emitters are not very big or heavy so I don't think it will affect build times much.

What do you guys think?